ecommerce/templates/oscar/basket/partials/hosted_checkout_basket.html

```diff
@@ -165,6 +165,8 @@
                             {% elif processor.NAME == 'paypal' %}
                                 {# Translators: Do NOT translate the name PayPal. #}
                                 {% trans "Checkout with PayPal" as tmsg %}{{ tmsg | force_escape }}
+                            {% else %}
+                                {% trans "Checkout" as tmsg %}{{ tmsg | force_escape }}
```

Comment:
@kaizoku, following up on the discussion [here](https://github.com/edx/ecommerce/pull/3297#discussion_r571967221),

If there are multiple additional payment processors configured using the changes in this PR, all of them will show `Checkout` in the button text and that is not very useful. So I'd say that allowing the custom payment processors to specify their own translatable text here would be a good thing and would only require adding a well-known property in the payment processor class. What do you say? This will be useful for the other payment processors that we may implement.

```suggestion
                            {% else %}
                                {% if processor.CHECKOUT_TEXT %}
                                    {{ processor.CHECKOUT_TEXT | force_escape }}
                                {% else %}
                                    {% trans "Checkout" as tmsg %}{{ tmsg | force_escape }}
                                {% endif %}
```
The `CHECKOUT_TEXT` property can be a translatable string and can be added as a property to the base payment processor class with `''` as the default value. What do you say?

CC @shimulch

Reply:
@kaizoku, I have fixed this in https://github.com/open-craft/ecommerce/pull/20/commits/9849b17a8d2633af59f9c865d798b0b2adda4896.

Reply:
@lgp171188 @kaizoku this change doesn't apply cleanly to lilac, and isn't present in the upstream PR: https://github.com/edx/ecommerce/pull/3297

Could you confirm if this is required?

Ticket: [FAL-2261](https://tasks.opencraft.com/browse/FAL-2261) (port code drift to lilac)